doc/toc.yml

```diff
@@ -1,115 +1,118 @@
 - name: Overview
-  href: Overview/ExtensionsOverviewAndGettingStarted.md
+  href: ExtensionsOverview.md
+- name: Frequently Asked Questions
+  href: Questions.md
+- name: Creating an Application
+  href: GettingStarted.md
+# - name: Overview topics (legacy)
+#   items:
+#     - name: Hosting
+#       href: Overview/Hosting/HostingOverview.md
```

Comment:
@lukeblevins where are all the overall topics now? I don't see them anywhere in the TOC?

Reply:
I thought that we decided to remove that section because it's incomplete?

Reply:
My bad, I will add that part back.

Reply:
I've integrated these topics back in a way which feels more intuitive and mirrors the general structure used by Microsoft. We now consolidate Learn and Overview into "Features"